### ISSUE TITLE
OpenStarbound has same goal but more activity

### ISSUE THREAD

**Issue by jkotrc** [rwf93/Starbound#13]
The repository https://github.com/OpenStarbound/OpenStarbound is another community project updating starbound. It has around 400 commits so far and far greater activity. Check it out

**Comment by rwf93** [comment]
Yeah, I more or less recommend people going to OpenStarbound as it seems to have more active development. I created this on the first day of the Starbound source code leak, and haven't worked committed to it as a full time project. I may actually pin this issue, or mention OpenStarbound in the readme.
